app/views/govuk_publishing_components/components/_feedback.html.erb

```diff
@@ -81,8 +92,19 @@
     </div>
   </form>
 
-  <form action="/contact/govuk/email-survey-signup" id="page-is-not-useful" class="gem-c-feedback__form gem-c-feedback__form--email gem-c-feedback__js-show js-feedback-form js-hidden" data-track-category="user_satisfaction_survey" data-track-action="banner_taken" method="post">
-    <a href="#" class="gem-c-feedback__close js-close-form" aria-controls="page-is-not-useful" aria-expanded="true" role="button">Close</a>
+  <form action="/contact/govuk/email-survey-signup"
+    id="page-is-not-useful"
+    class="gem-c-feedback__form gem-c-feedback__form--email gem-c-feedback__js-show js-feedback-form js-hidden"
+    data-track-category="user_satisfaction_survey"
+    data-track-action="banner_taken"
```

Comment:
The action `banner_taken` seems wrong now that I'm looking at it. It uses the same action as the banner, perhaps we'd like to differentiate surveys taken from the banner and the feedback component?